test/v3/controllers/requestParsers/validators/AmendBFLossValidatorSpec.scala

```diff
@@ -68,17 +68,10 @@ class AmendBFLossValidatorSpec extends UnitSpec {
           RuleIncorrectOrEmptyBodyError)
       }
     }
-    "return a FORMAT_LOSS_AMOUNT error" when {
+    "return a FORMAT_VALUE error" when {
       "a lossAmount greater than 2 decimal places is submitted" in {
-        validator.validate(amendBFLossRawData(validNino, validLossId, invalidLossAmountFormat)) shouldBe List(AmountFormatError)
-      }
-    }
-    "return a RULE_LOSS_AMOUNT error" when {
-      "a lossAmount less than the minimum allowed value is submitted" in {
-        validator.validate(amendBFLossRawData(validNino, validLossId, invalidLossAmountRuleUnderMin)) shouldBe List(RuleInvalidLossAmount)
```

Comment:
Please remove unused `invalidLossAmountRuleUnderMin` and unused `invalidLossAmountRuleOverMax` vals.